salt/cloud/libcloudfuncs.py

```diff
@@ -336,7 +341,7 @@ def reboot(name, conn=None):
         __utils__["cloud.fire_event"](
             "event",
             "{} has been rebooted".format(name),
-            "salt-cloud" "salt/cloud/{}/rebooting".format(name),
+            "salt/cloud/{}/rebooting".format(name),
```

Comment:
Not sure what's going on here... but it doesn't look right

Reply:
yes, this is correct.